Makefile

```diff
@@ -21,13 +24,22 @@ docs: ## generate Sphinx HTML documentation, including API docs
 	$(IN_VENV) $(MAKE) -C doc clean
 	$(IN_VENV) $(MAKE) -C doc html
 
+docs-schema-ready: ## Build Github-flavored Markdown from Galaxy Tool XSD
```

Comment:
These aren't generated files - they always exist in the repo so I wouldn't typically depend on them. It just as much depends on the Python script as well. So I'd prefer not to make this change if that is okay?